core/src/main/resources/application.conf

```diff
@@ -0,0 +1,106 @@
+collector {
+  enableDefaultRedirect = false
+
+  paths {
+
+  }
+
+  crossDomain {
+    enabled = false
+    domains = [ "*" ]
+    secure = true
+  }
+
+  cookieBounce {
+    enabled = false
+    name = "n3pc"
+    fallbackNetworkUserId = "00000000-0000-4000-A000-000000000000"
+  }
+
+  cookie {
+    enabled = true
+    expiration = 365 days
+    name = sp
+    domains = [
+
+    ]
+    fallbackDomain = "acme1.com"
+    secure = true
+    httpOnly = false
+    sameSite = "None"
```

Comment:
Please @paulboocock can you confirm these are sensible default values for cookies?  Only used if the user does not override them.

Reply:
These settings are the best defaults for modern browsers, and have the widest support for locations where you would track (such as cross domain, in an iframe, etc.). [Some older browsers](https://www.chromium.org/updates/same-site/incompatible-clients) reject `SameSite = None` cookies, so there is an argument for `Lax` being a better default but that wouldn't work cross domain. In reality, there isn't a great default for `sameSite` but I think for a server set cookie, None is probably the safest default.

As for other fields, `httpOnly` should probably be true by default. There's no reason to read this cookie client side, so not much reason to make it false. `secure` must be true is `SameSite = None`, so thats a good default.